cfme/tests/containers/test_cockpit.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from cfme.containers.node import Node, NodeCollection
+from cfme.containers.provider import ContainersProvider
+from cfme.web_ui import CheckboxTable, toolbar as tb
+from utils import testgen, version
+from utils.appliance.implementations.ui import navigate_to
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda provider: version.current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    [ContainersProvider], scope='function')
+
+
+@pytest.mark.polarion('CMP-10255')
+def test_cockpit_button_access(provider, soft_assert):
+    """ The test verifies the existence of cockpit "Web Console"
+        button on each node
+
+    """
+
+    navigate_to(NodeCollection, 'All')
+    tb.select('List View')
```

Comment:
This table is modeled already in the NodeView, no need to create it again.  This is why we're moving to thoroughly modeling all UI elements as widgets - to access them through the views generated by navigation.

The 'List View' selection is redundant - the 'All' destination currently calls it in resetter, which is called by default.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/node.py#L34

```python
# might need slight tweaks but this is how you would use the view and widgets after navigating
view = navigate_to(NodeCollection, 'All')

names = [r.name.text for r in view.nodes]
```

`view.nodes`, the widgetastic_manageiq.Table class, returns rows() through `__iter__`